Makefile

```diff
@@ -26,8 +26,9 @@ ifeq ($(IMAGE_NAME),)
 IMAGE_NAME = $(REGISTRY)/$(DRIVER_NAME)
 endif
 
-BUILDIMAGE_TAG ?= golang$(GOLANG_VERSION)
-BUILDIMAGE ?= $(IMAGE_NAME)-build:$(BUILDIMAGE_TAG)
+GO_VERSION := $(shell awk '/^go /{print $$2}' go.mod|head -n1)
+BUILDIMAGE_TAG ?= devel-go$(GO_VERSION)
+BUILDIMAGE ?=  ghcr.io/nvidia/k8s-test-infra:$(BUILDIMAGE_TAG)
```

Comment:
Add `BUILDIMAGE ?= ghcr.io/nvidia/k8s-test-infra:golang$(GOLANG_VERSION)` to `versions.mk` directly.